docs/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Build:
+# docker build -t ace3mod/jekyll .
+#
+# Run:
+# docker run -p 4000:4000 -it ace3mod/jekyll
+
+FROM starefossen/github-pages:onbuild
+MAINTAINER bux
+
+COPY . /usr/src/app
+WORKDIR /usr/src/app
```

Comment:
same as below, switch to latest, this is already done.

If not, the `COPY` on line 10 is unncessary.